### PR TITLE
Populatedb Use EndpointResourcesList.json

### PR DESCRIPTION
**Pull request by emichaud998**
Changes in this PR:

Instead of hardcoding the endpoint list filenames and source names (right now Cerner and Epic) in make populatedb, we now use the newly created EndpointResourcesList.json, which allows for more scalability

Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: [Lantern-236 JIRA Ticket](https://oncprojectracking.healthit.gov/support/browse/LANTERN-236)
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

Create a new branch `<branch name>_gomodupdate`. Run `make update_mods` to update the `go.mod`
and `go.sum` files to point to master. Create a PR. Require one sanity check reviewer.

### CODE REVIEW

File: scripts/populatedb.sh [hunk 1]
```diff
@@ -4,10 +4,15 @@ set -e
...
-# go run main.go /etc/lantern/resources/CareEvolutionEndpointSources.json CareEvolution
-go run main.go /etc/lantern/resources/CernerEndpointSources.json Cerner
-go run main.go /etc/lantern/resources/EpicEndpointSources.json Epic
+
+jq -c '.[]' /go/src/app/resources/EndpointResourcesList.json | while read endpoint; do
+    NAME=$(echo $endpoint | jq -c -r '.EndpointName')
+    FILENAME=$(echo $endpoint | jq -c -r '.FileName')
+
+    # Only use the line below that populates the database with CareEvolution for development 
+    # go run main.go /etc/lantern/resources/CareEvolutionEndpointSources.json CareEvolution
```
Comment (mayerm94) [inline scripts/populatedb.sh:13]: Move these outside the while loop
